frontend/src/views/Pages/Login.vue

```diff
@@ -104,13 +104,8 @@ export default {
           this.$router.push('/overview')
           loader.hide()
         })
-        .catch((error) => {
-          if (error.message.includes('No user with this credentials')) {
-            this.$toasted.global.error(this.$t('error.no-account'))
-          } else {
-            // : this.$t('error.no-email-verify')
-            this.$router.push('/reset/login')
-          }
+        .catch(() => {
+          this.$toasted.global.error(this.$t('error.no-account'))
```

Comment:
We need the push to /reset/login in case the user has not activated his account yet.
```suggestion
        .catch((error) => {
          this.$toasted.global.error(this.$t('error.no-account'))
          if (error.message.uncludes('User email not validated')) {
            this.$router.push('/reset/login')
          }
```

Reply:
it may be better to check for locales ... if the text should ever change ... for whatever reason :)  
      ```
       if (error.message.uncludes(this.$t('error.no-account'))) {
            this.$router.push('/reset/login')
       }
       ```